src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -40,6 +40,7 @@ public class Win32PlatformOptions
     {
         public bool UseDeferredRendering { get; set; } = true;
         public bool AllowEglInitialization { get; set; }
+        public bool? EnableMultitouch { get; set; }
```

Comment:
Why does this need to be an option?

Reply:
Since the feature is new, it can potentially break some existing controls that aren't ready for multiple pointers co-existing in the application. On the other hand, when we don't announce touch support, we still get emulated mouse events. That's why I've made this feature an opt-in for now.